camus-shopify/script/reconcile.sh

```diff
@@ -0,0 +1,36 @@
+CHECK_DATE=$1   # e.g. `date -d "yesterday" '+%Y/%m/%d'`  # e.g. 2017/01/01
+
+TIME_NOW=`date +%s`
+EXIT_CODE=0
+
+while read TOPIC; do
```

Comment:
where is TOPIC coming from?

Reply:
https://github.com/Shopify/camus/blob/769f42733c952bd7f8021c9c25fcf1ee65096ebe/camus-shopify/script/reconcile.sh#L34 
It's the same file we use to list the topics that need to be uploaded.
Confusing, because bash.

Reply:
ah, sweet, thanks.